lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -143,8 +171,10 @@ window.listenForStatus = function(callback) {
   log.events.addListener('status', callback);
 };
 
-chrome.runtime.onInstalled.addListener(details => {
-  if (details.previousVersion) {
-    console.log('previousVersion', details.previousVersion);
-  }
-});
+if (window.chrome) {
```

Comment:
`window.chrome` exists in Chrome typically. adding `&& chrome.runtime` would work for me.